src/lib/reducer.js

```diff
@@ -115,12 +115,12 @@ function reduceAddSubExpression(node, precision) {
     // (something + value) + value => something + (value + value)
     // (something - value) + value => something - (value + value)
```

Comment:
by the way this is also false, it should be:
(something - valueA) + valueB => something - (valueA - valueB)